guides/echo-guides/otp.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: 'How to send OTP verification email notifications with React-email'
+description: 'Learn how to send OTP verification email notifications with React-email in your NextJS app'
+---
+
+## Introduction
+
+In this guide, you’ll learn how to send OTP verification email notifications using the React-email package. Follow these steps:
+
+## Getting started
+
+Integrating Novu’s code-first workflow with React.Email for your Next.js application can be done in three steps
+
+1. Create a NextJS app and wait for the installation:
+
+```bash
+npx create-echo-app
+
+```
+
+2. Once this installation is complete, simply `cd` into the directory and start your app using the npm run dev command, and your app will be served on `localhost:4000`
```

Comment:
`and your app will be served on `localhost:4000` `

please remove that sentence because their app can be served on any port depending on how their system runs it or if they have another app running.

Reply:
`npm run dev` command should have back ticks so that it can show on the page like a command instead of a normal group of words.